Readme.md

```diff
@@ -27,12 +27,12 @@ import { Chat } from "@empirica/chat";
 
 ### Multiple chat instances within the same scope
 
-You can pass an optional `key` string prop to differenciate different chats
-within the same scope. This changes which key on the given scope the chat will
+You can pass an optional `customKey` string prop to differenciate different chats
+within the same scope. This changes which customKey on the given scope the chat will
```

Comment:
maybe call it "get/set key" in this context.

Reply:
something like this "You can get/set key as `customKey` string ...... " ?

Reply:
```suggestion
within the same scope. This changes which get/set key on the given scope the chat will
```